tests/workload/loader_test.py

```diff
@@ -1942,7 +1942,7 @@ def test_parse_duplicate_implicit_task_names(self):
         reader = loader.WorkloadSpecificationReader()
         with self.assertRaises(loader.WorkloadSyntaxError) as ctx:
             reader("unittest", workload_specification, "/mappings")
-        self.assertEqual("Workload 'unittest' is invalid. TestProcedure 'default-test_procedure' contains multiple tasks"
+        self.assertEqual("Workload 'unittest' is invalid. Procedure 'default-test_procedure' contains multiple tasks"
```

Comment:
`default-test_procedure` will also be corrected when you do a replace for `test_procedure` -> `procedure`